components/usage/pkg/db/cost_center.go

```diff
@@ -38,12 +38,14 @@ func (d *CostCenter) TableName() string {
 	return "d_b_cost_center"
 }
 
-type DefaultSpendingLimit struct {
-	ForTeams int32 `json:"forTeams"`
-	ForUsers int32 `json:"forUsers"`
+type DefaultUsageLimits struct {
+	ForTeams       int32 `json:"forTeams"`
+	ForUsers       int32 `json:"forUsers"`
+	ForStripeTeams int32 `json:"forStripeTeams"`
+	ForStripeUsers int32 `json:"forStripeUsers"`
```

Comment:
What's the difference between Stripe and nonStripe users?

Would it make sense to introduce a child struct which holds the values for User/Teams like this?
```
type Limit struct {
  Team int32
  User int32
}

and use it as 

type DefaultUsageLimits {
  WithoutSubscription Limit
  WithSubscription Limit
}
```